apis/dns/v1beta2/zz_recordset_types.go

```diff
@@ -188,7 +188,7 @@ type HealthCheckedTargetsInternalLoadBalancersParameters struct {
 
 	// The type of load balancer. This value is case-sensitive. Possible values: ["regionalL4ilb", "regionalL7ilb", "globalL7ilb"]
 	// +kubebuilder:validation:Optional
-	LoadBalancerType *string `json:"loadBalancerType" tf:"load_balancer_type,omitempty"`
+	LoadBalancerType *string `json:"loadBalancerType,omitempty" tf:"load_balancer_type,omitempty"`
```

Comment:
We should backport the changes in this file.

Reply:
Done. I've seen inconsistencies in generating of `omitempty` tags. Either there's an edge case that I don't know about or there's a bug that omits generating them. If it's because of a bug and we fix it, we don't need any modifications to our backporting procedure, i.e., we don't need to backport struct tags.

Reply:
Adding `omitempty` tags didn't result in any changes in `make generate`, which was obvious to me only in hindsight 🙂